conduit/plugins/plugin.go

```diff
@@ -0,0 +1,16 @@
+package plugins
+
+// Plugin is the common interface for all Conduit plugins.
+type Plugin interface {
```

Comment:
We can add `Init()` to this interface if we add `GetGenesis()` to `Importer`'s interface and do some refactoring around the receivers of `Importer.Init()`

Reply:
Intriguing. This is something I tried to do but couldn't quite pull off. I hadn't considered splitting `GetGenesis` into a separate function.

I believe `InitProvider` has a `SetGenesis` function. The importer could be required to call that (and we generate an error in the pipeline code if it doesn't).

Reply:
Are you suggesting adding both:
* `GetGenesis()`
* `SetGenesis()`
to the `Importer` interface?

Reply:
Only added `GetGenesis()`